content/_data/changelogs/weekly.yml

```diff
@@ -11142,10 +11134,69 @@
   # pull: 5501 (PR title: Bump bridge-method-annotation from 1.18 to 1.21)
   # pull: 5502 (PR title: Bump spotless-maven-plugin from 2.11.0 to 2.11.1)
 
-- version: '2.295'
+- version: "2.295"
   date: 2021-05-25
   changes: []
 
+- version: "2.296"
+  date: 2021-06-01
+  changes:
+    - type: major bug
+      category: regression
+      pull: 5479
+      issue: 65585
+      authors:
+        - thomasgl-orange
+        - basil
+      message: |-
+        Fix regressions in forms submissions from unwanted form validation in any browser (regression in 2.289).
+    - type: rfe
```

Comment:
@MarkEWaite Do we want this "rfe" change to be below the "bug" changes?

Reply:
Good question.  The [style guide "order of entry" section](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/_STYLEGUIDE.adoc#order-of-entries-regular) suggests alternating between blocks of bug and and blocks of RFE so that the color difference helps visually separate groups of entries

> If in doubt, alternate rfe and bug entries to use the contrasting color to visually separate groups of entries.

Based on that, I think we want to keep the ordering as it is so that the "major bugs" block is followed by a block (of one) for RFE and then a block for "bugs".

Reply:
I totally got your point, thank you for your detailed explanation.